scripts/claude_start.sh

```
@@ -0,0 +1,1734 @@
+if [ -f "\$BACKUP_SCRIPT" ]; then
+    cd "\$MEMORY_BACKUP_REPO"
+    python3 "\$BACKUP_SCRIPT" --mode=cron
+else
+    echo "\$(date): Unified memory backup script not found at \$BACKUP_SCRIPT" >> /tmp/memory_backup_errors.log
+fi
+EOF
+
+        chmod +x "$CRON_WRAPPER"
+
+        # Add to cron (daily at 2 AM) with proper variable handling
+        local current_crontab_mem
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)